index.html

```diff
@@ -80,163 +81,176 @@ <h3 id="Background">Background</h3>
-								<li><code>windows-1252;</code></li>
+								<li><code>windows-1252;</code> (On Windows)</li>
 							</ol>
 							<li><code>CSV Schema Encoding</code>, which has a dropdown menu giving the choice of:</li>
 							<ol>
 								<li><code>UTF-8</code> and;</li>
-								<li><code>windows-1252;</code></li>
+								<li><code>windows-1252;</code> (On Windows)</li>
 							</ol>
-							<li><code>Fail on first error?</code>;</li>
-							<li><code>Enforce case-sensitive filepath checks?</code>;</li>
+							<li><code>Maximum number of errors to display</code>;</li>
+							<li><code>Maximum number of characters in a cell</code>;</li>
+							<li><code>Fail on first error</code>;</li>
+							<li><code>Enforce case-sensitive filepath checks</code>;</li>
 							<li><code>Validate CSV for valid UTF-8 characters</code>, and;</li>
 							<li><code>Path Substitutions</code></li>
 						</ol>
 					</p>
-					<img src="images/1.3.0/settings-opened.png" alt="Part of GUI, showing opened Settings section" id="GUI-detail5">
+					<img src="images/1.4.0/settings-opened.png" alt="Part of GUI, showing opened Settings section" id="GUI-detail5">
 					<div>
+						<h4>Maximum number of errors to display</h4>
+						<p>This setting will determine the number of lines of errors that are output in the output pane; it is set to 2000 by default.</p>
+						<h4>Maximum number of characters in a cell</h4>
+						<p>
+							This setting will set a limit on the number of characters allowed in a cell; the default number of characters is 4096
+						</p>
 						<h4>Fail on first error</h4>
 						<p>
 						With this setting selected, validation will terminate when the first 
-						<a href="http://digital-preservation.github.io/csv-schema/csv-schema-1.0.html#validation-errors">Validation Error</a> is encountered in the CSV file.  
-						If there are no validation errors the whole CSV file will be validated and return <code>PASS</code>.
+						<a href="https://digital-preservation.github.io/csv-schema/csv-schema-1.0.html#validation-errors">validation error</a> is encountered in the CSV file.
+						If there are no validation errors, the whole CSV file will be validated and return <code>PASS</code>.
 						You should be aware that the CSV Schema can declare that some validation tests will produce only validation warnings, rather than validation errors. 
 						A validation warning will not stop validation, even with "Fail on first error" set.
-						This setting is equivalent to using the -f true or --failfast true options when <a href="#command-line-tool">running via the Command Line</a>.
+						This setting is equivalent to using the <code>-f true</code> or <code>--failfast true</code> options when <a href="#command-line-tool">running via the Command Line</a>.
 						</p>
 						Note: If you are working with a particularly large file, it can be beneficial to work with this option enabled. If there are errors that are likely to
```

Comment:
I think this entire `Note` needs to be enclosed in `<P></P>` tag, so that there is a line break before the next section